src/sparseml/exporters/transforms/convert_quantizable_conv_integer.py

```diff
@@ -125,22 +69,23 @@ def convert_conv_to_quantized(
         weight_quantize_params=get_quantization_params(
             model, weight_quantize_node, include_target=True
         ),
-        bias_initializer=graph.get_init_by_name(conv_node.input[2]),
-        bias_add_name="{}_bias_add".format(conv_node.name),
-        target_output=conv_node.output[0],
+        bias_initializer=bias_init,
+        bias_add_name="{}_bias_add".format(match.node.name),
+        target_output=match.node.output[0],
         transpose_weight=False,
     )
 
     delete_quant_node(model, weight_dequantize_node)
     delete_quant_node(model, weight_quantize_node)
+    delete_quant_node(model, input_dequantize_node)
```

Comment:
Watch out, this duplicates line 85

Reply:
That is deleting the quantize node it looks like? whereas this is the dequant?

I've seen some quantize nodes have multiple children, which would mean we can't delete it without breaking the graph

Reply:
Perhaps the delete_quant_node should check that there are no more usages of the node before deleting?